source/configuration_manual/imap_metadata.rst

```diff
@@ -25,3 +25,53 @@ Example:
   protocol imap {
     imap_metadata = yes
   }
+
+Storing metadata in SQL dictionary
+==================================
+
+You can store metadata into a database too. This works best with dedicated table
+for storing the entries. 
+
+Database schema
+---------------
+
+.. code:: sql
+
+  CREATE TABLE metadata (
+    attr_name VARCHAR(255) NOT NULL,
+    username VARCHAR(255) NOT NULL DEFAULT '',
+    attr_value VARCHAR(65535),
```

Comment:
* Would be prettier sorted, maybe username first and then attr_name & attr_value
 * DEFAULT '' for username sounds dangerous. I don't think username should ever be ''

Reply:
It has to have some default value, primary keys cannot be NULL.

Reply:
So ignore the above, also the order. But add some note about using empty values for shared attributes.